snowpack/src/build/build-import-proxy.ts

```diff
@@ -75,7 +75,12 @@ export function wrapHtmlResponse({
   });
 
   if (hmr) {
-    const hmrScript = `<script type="module" src="${getMetaUrlPath('hmr-client.js', config)}"></script><script type="module" src="${getMetaUrlPath('hmr-error-overlay.js', config)}"></script>`;
+    let hmrScript = ``;
+    if (mode === 'production' && config.devOptions.hmrPort) {
```

Comment:
It's not a good way to check whether `mode` is production. need your suggestions.

Reply:
Why only `production`? Does this not work in dev mode?

Reply:
bump! would love an answer so that we can resolve and merge!

Reply:
@FredKSchott sorry, I've been very busy recently. 

> Why only production? Does this not work in dev mode?
It's a good problem. I think `hmrPort` should work in dev mode 🤣.

Reply:
@FredKSchott I made a fix and the `hmrPort` option would take effect in dev mode. I manually test 6 cases above.